Prace_domowe/Praca_domowa3/Pawel_Wojciechowski/pd3-Pawel_Wojciechowski.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>Zasadniczo jest ok.</p><p> </p><p>Przydałoby się porównać wyniki na treningowym i testowym, żeby sprawdzić czy nie mamy overfittingu.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Chyba LR ma większy recall niż drzewo, a może to jest najlepsza metryka? Ludzie nie lubią, gdy pada, a nie mają parasolki</p>

Reply:
<p>Jak wyniki są tak podobne, to tabelka też jest mile widziana</p>